hooks/basic/collector.py

```diff
@@ -191,6 +191,12 @@ def settings_schema(self):
                     "fields": "context, *",
                     "allows_empty": True,
                 },
+                "resolution_order": {
+                    "type": "int",
+                    "default_value": 0,
+                    "allows_empty": True,
+                    "description": "Resolution order to follow in case of conflicts."
```

Comment:
Can you explain in more detail how this works and why its necessary?

Reply:
I will update it in the description as well.

This setting will enable us to configure which item type gets priority in case of multiple item types end up with no matching, technically we want to give preference to an item type that doesn't have a work_path_template set in that scenario, so rest of our context and template resolution works using `template_from_path`